pkg/executor/scraper/scraper.go

```diff
@@ -14,43 +10,22 @@ type Scraper interface {
 	Scrape(executionID string, directories []string) error
 }
 
-// NewMinioScraper returns a Minio implementation of the Scraper
-func NewMinioScraper(endpoint, accessKeyID, secretAccessKey, location, token, bucket string, ssl bool) *MinioScraper {
-
-	return &MinioScraper{
-		Endpoint:        endpoint,
-		AccessKeyID:     accessKeyID,
-		SecretAccessKey: secretAccessKey,
-		Location:        location,
-		Token:           token,
-		Bucket:          bucket,
-		Ssl:             ssl,
-	}
-
+type ScraperV2 struct {
```

Comment:
I think we should discuss versioning conventions (V2) usually in go there is subdirectory for V2

And do we need V2 here? As looks like you're overriding existing one

In This example I don't think we need V2 at all
Just from Testkube version 1.10 (some minor bump) we'll allow to handle Cloud + MinIO

Reply:
If we introduce V2, then we need to have V1 as well. If you just refactor the package, you can keep existing version

Reply:
keep in mind I need to refactor every single executoro